### PR TITLE
Add --all flag to volume rm

### PR DESCRIPTION
**Pull request by savage-engineer**
storageos volume rm --all will request that all volumes are deleted, if
a volume is mounted it will be passed over.
storageos volume rm --all -f will delete all volumes (including those
that are mounted)

Addresses #76

### CODE REVIEW

File: cli/command/volume/remove.go [hunk 1]
```diff
@@ -27,22 +28,54 @@ func newRemoveCommand(storageosCli *command.StorageOSCli) *cobra.Command {
...
+		}
+
+		if len(volumes) == 0 {
+			fmt.Fprintf(storageosCli.Err(), "%s\n", "no volumes to delete")
+			status = 1
+		} else {
+			opt.volumes = make([]string, len(volumes))
+			for i, volume := range volumes {
+				ref := volume.Namespace + "/" + volume.Name
+				opt.volumes[i] = ref
```
Comment (rusenask) [inline cli/command/volume/remove.go]: why add it if force is not supplied?

Comment (croomes) [reply cli/command/volume/remove.go]: force is only needed if the volume is mounted

Comment (rusenask) [reply cli/command/volume/remove.go]: ah yes

**Comment by croomes** [comment]
Works as I would expect:
```
simon@nimrod:~$ s ns create simon                   
simon                                               
simon@nimrod:~$ s v create simon/test01             
simon/test01                                        
simon@nimrod:~$ s ns create croome                  
croome                                              
simon@nimrod:~$ s v create croome/test01                                                                                                                                                                        
croome/test01                                       
simon@nimrod:~$ s v ls                              
NAMESPACE/NAME                                     SIZE                MOUNTED BY          NODE SELECTOR       STATUS              REPLICAS            LOCATION                                                 
default/pvc-9d80b9d7-004f-11e8-8720-38d54779ceff   1GB                 nimrod                                  active              0/0                 nimrod (healthy)                                         
simon/test01                                       5GB                                                         active              0/0                 nimrod (healthy)                                         
croome/test01                                      5GB                                                         active              0/0                 nimrod (healthy)                                         
simon@nimrod:~$ s v rm --all                        
simon/test01                                        
API error (Precondition Failed): cannot delete mounted volume 'default/pvc-9d80b9d7-004f-11e8-8720-38d54779ceff'                                                                                                
croome/test01                                       
simon@nimrod:~$ s v ls
NAMESPACE/NAME                                     SIZE                MOUNTED BY          NODE SELECTOR       STATUS              REPLICAS            LOCATION
default/pvc-9d80b9d7-004f-11e8-8720-38d54779ceff   1GB                 nimrod                                  active              0/0                 nimrod (healthy)
simon@nimrod:~$ 
```
Think we should merge.  There is a PR for the E2E tests also ready to merge.

**Comment by savage-engineer** [comment]
@JoeReid as it currently functions, running `storageos volume rm --all` will remove all volumes not mounted across all namespaces.

**Comment by croomes** [comment]
closes #76

**Pull request merged**